index.html

```diff
@@ -2371,6 +2371,24 @@ <h2>Secure Practices for designing levels below the WoT Runtime</h2>
       <section id="sec-pract-end-to-end-security">
         <h2>Secure Practices for end-to-end security</h2>
             <p>
+		    The notion 'end-to-end' security is often encountered when security is considered. 
+		    This notion is often misleading: it is a matter of risk assessment to determine 
+		    the "ends" that are to be protected (identifying the domain-of-protection) and 
+		    then select one or more security technologies to fulfill the identified demand 
+		    (rather than vice versa). In that sense, there is no single, universal notion of
+		    "end-to-end" in WoT security. There is also no single security technology that 
+		    allows to deliver "end-to-end security" for any definition of the ends. The 
+		    perception of WoT security is to support users of WoT security in implementing 
+		    security architectures according their perception of the ends (which can vary 
+		    across users, deployments etc.)
+		    For instance: MACSec provides E2E security between endpoints in one (local) 
+		    network, IPSec provides E2E security between endpoints in an IP-based internetwork, 
+		    (D)TLS provides E2E security between client and server processes, CMS as well as 
+		    XML/JSON/CBOR security can provide E2E security on level of application data
+            </p>
+	    <p>
+		    The following text addresses E2E security in the sense of protecting application 

```

Comment:
I would change "text" to "technologies", in that these are examples of how to support certain forms of protection for certain definitions of "ends".  Maybe some of the above (MACSec, etc.) can be moved to a common "examples" sub-section (not a section, but perhaps a bulleted list?).  Each example should define what the "ends" are and what is being protected.

Reply:
In short, we need to integrate the existing text into the discussion, not end with "and here's some other stuff"